.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: "CI"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  phpunit:
+    name: "PHP ${{ matrix.php }}"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+          - "8.0"
+          - "8.1"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php }}"
+          coverage: none
+          tools: pecl, composer:v1
```

Comment:
```suggestion
          tools: pecl, composer:v2
```

Reply:
Would like to keep `v1` or you forget it?

Reply:
Ahh good catch, just forgot it. Fixed.